services/gateway/security/jwtUtils.go

```diff
@@ -94,6 +96,9 @@ func getPrivateKey(keyEnv *appConfig.EnvInlineOrPath) (interface{}, error) {
 	}
 
 	block, _ := pem.Decode(pemFile)
+	if block == nil {
+		return nil, errors.New("no PEM found")
+	}
 	key, err := x509.ParsePKCS8PrivateKey(block.Bytes)
```

Comment:
can do the casting of RsaKey like here: https://github.com/youmark/pkcs8/blob/master/pkcs8.go#L116 and then return *rsa.PrivateKey instead of interface{}